lib/puppet/transaction/persistence.rb

```diff
@@ -56,18 +56,21 @@ def load
       begin
         result = Puppet::Util::Yaml.load_file(filename)
       rescue Puppet::Util::Yaml::YamlLoadError => detail
-        Puppet.err "Transaction store file #{filename} is corrupt (#{detail}); replacing"
+        Puppet.log_exception(detail, "Transaction store file #{filename} is corrupt (#{detail}); replacing")
```

Comment:
Should this be using the new "log exceptions as a warning" machinery that I added?